sdk/core/Azure.Core/tests/RehydrationTokenTests.cs

```diff
@@ -18,7 +18,7 @@ public void ThrowOnDeserializationWithNullRehydrationToken()
         [Test]
         public void ThrowOnDeserializationWithRehydrationTokenNullRequiredMember()
         {
-            var data = BinaryData.FromString("\"requestMethod\": null}");
+            var data = BinaryData.FromString("{\"requestMethod\": null}");
```

Comment:
Would it make sense to validate the exception type here?

Reply:
It throws at [Debug.Assert](https://github.com/Azure/azure-sdk-for-net/blob/7bf98778e796c6c6c7f77482f6fb04b5f107ea7a/sdk/core/Azure.Core/src/RehydrationToken.Serialization.cs#L76), the exception type is `Microsoft.VisualStudio.TestPlatform.TestHost.DebugAssertException`, which is internal, not assertable for testing.